src/core/query/woql_compile.pl

```diff
@@ -5309,3 +5311,41 @@
     test_lit(duration(-1,0,0,0,1,0,0)^^xsd:duration, "\"-PT1H\"^^'http://www.w3.org/2001/XMLSchema#duration'").
 
 :- end_tests(store_load_data).
+
+:- begin_tests(preflight).
+:- use_module(core(util/test_utils)).
+:- use_module(core(api)).
+:- use_module(core(query)).
+:- use_module(core(triple)).
+:- use_module(core(transaction)).
+
+test(json_unbound_capture, [
```

Comment:
Is this related to the rest of the PR or an accidental inclusion?

Reply:
Fixed naming